sematic/db/migrations/20221208183307_create_server_settings_file.py

```diff
@@ -2,30 +2,29 @@
 import os
```

Comment:
This migration file was already released in 0.21.0 and 0.21.1. It cannot be modified.
Please add another migration file on top which does these changes.